src/nupic/algorithms/KNNClassifier.py

```diff
@@ -259,6 +259,7 @@ def _sparsifyVector(self, inputPattern, doWinners=False):
     if doWinners:
       if (self.numWinners>0) and (self.numWinners < (inputPattern > 0).sum()):
         sparseInput = numpy.zeros(inputPattern.shape)
+        
```

Comment:
Do you use Eclipse? Because the Python plugin doesn't have the capacity to show invisible chars, so how do you see these? (The Java plugin does have a way to see these chars)

Please tell me how?